src/metabase/automagic_dashboards/core.clj

```diff
@@ -1138,7 +1149,7 @@
 (defn cell-title
   "Return a cell title given a root object and a cell query."
   [root cell-query]
-  (str/join " " [(if-let [aggregation (qp.util/get-in-normalized (-> root :entity) [:dataset_query :query :aggregation])]
+  (str/join " " [(if-let [aggregation (get-in (-> root :entity) [:dataset_query :query :aggregation])]
```

Comment:
:entity could be moved into the keys vector

Reply:
oh yeah I'll fix that